src/Autofac.Extras.AggregateService/Autofac.Extras.AggregateService.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Dynamic aggregate service implementation generation for Autofac.</Description>
     <!-- VersionPrefix patched by AppVeyor -->
     <VersionPrefix>0.0.1</VersionPrefix>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0; netstandard2.1</TargetFrameworks>
```

Comment:
I thought it'd good to align this with the Autofac package. What do you guys think?

Reply:
Agreed.